c2rust-analyze/src/main.rs

```diff
@@ -742,7 +742,7 @@ fn fn_body_owners_postorder(tcx: TyCtxt) -> Vec<LocalDefId> {
 
         match tcx.def_kind(root_ldid) {
             DefKind::Fn | DefKind::AssocFn => {}
-            DefKind::AnonConst => continue,
+            DefKind::AnonConst | DefKind::Const => continue,
```

Comment:
What does this fix?

Reply:
being able to handle things like `pub const HANDLER_GO_ON: handler_t = 0;` without associating a body with them